SimPEG/EM/NSEM/survey.py

```diff
@@ -1,86 +1,84 @@
-from __future__ import print_function
-from __future__ import absolute_import
-from __future__ import division
-
 import sys
 import numpy as np
 from numpy.lib import recfunctions as recFunc
 
-from SimPEG import Survey as SimPEGsurvey, mkvc
-from .SrcNSEM import BaseNSEMSrc, Planewave_xy_1Dprimary, Planewave_xy_1DhomotD
-from .RxNSEM import Point_impedance3D, Point_tipper3D
+from ..FDEM.survey import Survey
+from ...data import Data as BaseData
+from ...utils import mkvc
+from .source import Planewave_xy_1Dprimary, Planewave_xy_1DhomotD
+from .receiver import Point_impedance3D, Point_tipper3D
 from .Utils.plotUtils import DataNSEMPlotMethods
 
 #########
 # Survey
 #########
 
 
-class Survey(SimPEGsurvey.BaseSurvey):
-    """
-    Survey class for NSEM.
+# class Survey(BaseSurvey):
+#     """
+#     Survey class for NSEM.
 
-    **Requried**
-    :param list srcList: List of sources associated with the survey
+#     **Requried**
+#     :param list srcList: List of sources associated with the survey
 
 
-    **Optional**
-    """
-    srcPair = BaseNSEMSrc
-
-    def __init__(self, srcList, **kwargs):
-        # Sort these by frequency
-        self.srcList = srcList
-        SimPEGsurvey.BaseSurvey.__init__(self, **kwargs)
-
-        _freqDict = {}
-        for src in srcList:
-            if src.freq not in _freqDict:
-                _freqDict[src.freq] = []
-            _freqDict[src.freq] += [src]
-
-        self._freqDict = _freqDict
-        self._freqs = sorted([f for f in self._freqDict])
-
-    @property
-    def freqs(self):
-        """Frequencies"""
-        return self._freqs
-
-    @property
-    def nFreq(self):
-        """Number of frequencies"""
-        return len(self._freqDict)
-
-    def getSrcByFreq(self, freq):
-        """Returns the sources associated with a specific frequency."""
-        assert freq in self._freqDict, "The requested frequency is not in this survey."
-        return self._freqDict[freq]
-
-    def eval(self, f):
-        """
-        Evalute and return Data given calculated fields
+#     **Optional**
+#     """
+#     srcPair = BaseNSEMSrc
 
-        :param SimPEG.EM.NSEM.FieldsNSEM f: A NSEM fileds object to evaluate data from
-        :retype: SimPEG.EM.NSEM.Data
-        :return: NSEM Data object
-        """
-        data = Data(self)
-        for src in self.srcList:
-            sys.stdout.flush()
-            for rx in src.rxList:
-                data[src, rx] = rx.eval(src, self.mesh, f)
-        return data
+#     def __init__(self, srcList, **kwargs):
```

Comment:
@grosenkj: This functionality is already in the base class, so I plan to remove it. Let me know if you have any concerns!